repose-aggregator/src/docs/asciidoc/release-notes.adoc

```diff
@@ -42,16 +42,16 @@ However, there were some internal contract changes with the Via and Location hea
 
 == 8.4.0.1
 
-* https://repose.atlassian.net/browse/REP-4795[REP-4795] https://repose.atlassian.net/browse/REP-4831[REP-4831] - the <<filters/saml-policy.adoc#, SamlPolicyTranslationFilter>> has been released!
-* https://repose.atlassian.net/browse/REP-4653[REP-4653] - the <<filters/rackspace-auth-user.adoc#, Rackspace Auth User filter>> updated to read request body of Forgot Password request to get the username and the <<filters/herp.adoc#, Highly Efficient Record Processor (HERP) filter>> was updated to get `X-User-Name` from response headers.
-* https://repose.atlassian.net/browse/REP-4928[REP-4928] - the <<filters/keystone-v2.adoc#, Keystone v2 filter>> will now return a 401 if self-validating tokens are being used and the Identity service responds with a 401.
+* https://repose.atlassian.net/browse/REP-4795[REP-4795] https://repose.atlassian.net/browse/REP-4831[REP-4831] - the <<filters/saml-policy.adoc#, SAML Policy Translation Filter>> has been released!
+* https://repose.atlassian.net/browse/REP-4653[REP-4653] - the <<filters/rackspace-auth-user.adoc#, Rackspace Auth User Filter>> updated to read request body of Forgot Password request to get the username and the <<filters/herp.adoc#, Highly Efficient Record Processor (HERP) Filter>> was updated to get `X-User-Name` from response headers.
+* https://repose.atlassian.net/browse/REP-4928[REP-4928] - the <<filters/keystone-v2.adoc#, Keystone v2 Filter>> will now return a 401 if self-validating tokens are being used and the Identity service responds with a 401.
 * https://repose.atlassian.net/browse/REP-4841[REP-4841] - a more unique ID will be used for User Access Events (UAE) in support of Cloud Auditing Data Federation (CADF).
-* https://repose.atlassian.net/browse/REP-4867[REP-4867] - the <<filters/valkyrie-authorization.adoc#, Valkyrie Authorization filter>> now supports multiple Character Encoding schemes.
+* https://repose.atlassian.net/browse/REP-4867[REP-4867] - the <<filters/valkyrie-authorization.adoc#, Valkyrie Authorization Filter>> now supports multiple Character Encoding schemes.
 * https://repose.atlassian.net/browse/REP-4954[REP-4954] - added support for Form Encoded requests (`Content-Type: application/x-www-form-urlencoded`).
 * https://repose.atlassian.net/browse/REP-4880[REP-4880] - internal utility classes JCharSequence and MessageDigester were removed.
 * https://repose.atlassian.net/browse/REP-4892[REP-4892] - versioned searching of these docs has been fixed.
-* https://repose.atlassian.net/browse/REP-4999[REP-4999]﻿- leading and trailing whitespace in directory values in the container.cfg.xml file are now ignored.
+* https://repose.atlassian.net/browse/REP-4999[REP-4999] - leading and trailing whitespace in directory values in the container.cfg.xml file are now ignored.
 
-== 8.3.0.1 (2016-12-13)
```

Comment:
Ok, I can add it back. I thought it would be duplication of effort with the documentation and real release notes page which both include this information already, but I'm cool either way.

![image](https://cloud.githubusercontent.com/assets/6766560/26589283/3c1bf06c-451d-11e7-8331-aea70f6b3113.png)